mesonbuild/wrap/wraptool.py

```diff
@@ -70,9 +70,13 @@ def list_projects(options: 'argparse.Namespace') -> None:
 def search(options: 'argparse.Namespace') -> None:
     name = options.name
     releases = get_releases()
-    for p in releases.keys():
-        if p.startswith(name):
+    for p, info in releases.items():
+        if p.find(name) != -1:
             print(p)
+        else:
+            for dep in info.get('dependency_names', []):
+                if dep.find(name) != -1:
+                    print("Dependency {dep} found in wrap {p}")
```

Comment:
I don't need a unittest to review this -- but it's not an f-string so this will not do variable replacement.